py_gtfs_rt_ingestion/tests/test_ingest.py

```diff
@@ -1,4 +1,11 @@
-from ingest import parseArgs
+from pathlib import Path
+import logging
+import pyarrow.parquet as pq
+
+from ingest import parseArgs, convert_json_to_parquet
+
+LOGGER = logging.getLogger(__name__)
+TEST_FILE_DIR = Path(__file__).parent.joinpath("test_files")
```

Comment:
where to find our test files. we might eventually want this to live in a helper that all the tests can see.